src/js/services/user-service.js

```diff
@@ -65,3 +67,30 @@ export function trackEvent(eventName, userProps) {
         return response;
     });
 }
+
+export function updateNowViewing(deviceId) {
+    if (!deviceUpdateThrottle) {
```

Comment:
I think it was by design, but maybe not. User is force-saved before sending a message anyway. But yeah might be a good idea to do the tail call.. Can you add it? pls